terraform/templates/container_definition.json.tmpl

```diff
@@ -24,6 +24,14 @@
       "name": "REDIS_ENDPOINT",
       "value": "${redis_endpoint}"
     },
+    {
+      "name": "FORMS_API_URL",
+      "value": "${forms_api_url}"
+    },
+    {
+      "name": "CONTROL_TOWER_URL",
```

Comment:
Can you change this to `AUTH_URL`
Control tower isn't used anymore and is just tech debt in the other repos.
Want to avoid it being mentioned in this one!